server/src/main/java/org/elasticsearch/cluster/SnapshotsInProgress.java

```diff
@@ -721,7 +732,7 @@ public Entry(
                 failure,
                 userMetadata,
                 version,
-                null,
+                (SnapshotId) null,
```

Comment:
Distinguishing overloads by the type of a single nullable parameter? Bleurgh.

Reply:
I know :) lets do https://github.com/elastic/elasticsearch/pull/89813 first then ;)